### ISSUE TITLE
Chapter 8/Run Example doesn't use ticks, but `go` waits on reset ticks

### ISSUE THREAD

**Issue by qiemem** [NetLogo/models#60]
It should probably just use ticks

**Comment by nicolaspayette** [comment]
Thanks for the report!

It's a duplicate of https://github.com/NetLogo/models/issues/50 and already fixed in the repo. It's also on my list of models to convert in https://github.com/NetLogo/models/issues/58.

**Comment by qiemem** [comment]
Damnit, I should really check that kind of thing first. Sorry about that!

**Comment by nicolaspayette** [comment]
I much prefer a duplicate report than no report at all! :-)
